### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by aakuptsov** [ctbrng/Tropnikova_ITproject#1]
1. Заявка оформлена не по правилам и не может быть зачтена

2. В приложение нужно добавить больше интерактива, функций. По сути, это что-то вроде Википедии, но визуализированной. Для дальнейшей работы над проектом нужно добавить возможность как-то изменять или влиять на "вселенную" приложения.

**Comment by ctbrng** [comment]
Здравствуйте! Насколько я поняла, приложение нельзя оставить в виде "Google maps", да? Тогда можно ли добавить какие-то мини-игры, пройдя которые, игрок сможет разблокировать новые комнаты/дома, или же обязательно нужно чтобы была возможность что-то менять во "вселенной" приложения? Просто изначально задумка была в том, что нигде не было достоверного изображения этого города, поэтому и хотелось создать что-то полностью отображающее мультсериал.

**Comment by aakuptsov** [comment]
![screen shot 2018-12-16 at 16 40 22](https://user-images.githubusercontent.com/24792175/50054300-76e1d080-0151-11e9-91e0-e5404277a5bc.png)


1. Сделай, пожалуйста, так, чтобы заявка выглядела вот таким образом, как на скриншоте выше.

2. Переименуй, пожалуйста, файл в соответствии с требованиями. Без этих двух пунктов заявка не может быть зачтена.

3. >> Насколько я поняла, приложение нельзя оставить в виде "Google maps", да?

Я, в свою очередь, не очень понял, откуда этот вопрос.

4. >>поэтому и хотелось создать что-то полностью отображающее мультсериал.

Сама по себе идея классная, думаю, поклонники сериала это оценят. Но я не совсем понимаю пока, где здесь программирование и что будет делать пользователь. 

5. >>можно ли добавить какие-то мини-игры, пройдя которые, игрок сможет разблокировать новые комнаты/дома

Это было бы весьма любопытно, особенно, если игры будут как-то связаны с персонажами сериала и их особенностям (например, любовь Гомера к пончикам, его работа на электростанции и т. п.)

6. >>или же обязательно нужно чтобы была возможность что-то менять во "вселенной" приложения?

Да, это тоже нужно. Это позволит освоить много всяких скиллов и знаний, которые потом пригодятся.

**Comment by ctbrng** [comment]
Да, прошу прощения за оформление - долго не могла разобраться, как это сделать. Сейчас все должно быть в порядке.
